jcommune-service/src/main/java/org/jtalks/jcommune/service/GroupService.java

```diff
@@ -53,7 +54,7 @@
      */
     List<Group> getByName(String name);
 
-    List<UserDto> getGroupUsers(long id, int count);
+    Page<UserDto> getPagedGroupUsers(long id, String page, int pageSize);
```

Comment:
It also makes sense to construct a PageRequest object in the controllers and pass that object here instead of primitives.